sklearn/datasets/descr/twenty_newsgroups.rst

```diff
@@ -220,4 +230,4 @@ the ``--filter`` option to compare the results.
 
    * :ref:`sphx_glr_auto_examples_model_selection_grid_search_text_feature_extraction.py`
 
-   * :ref:`sphx_glr_auto_examples_text_plot_document_classification_20newsgroups.py`
+   * :ref:`sphx_glr_auto_examples_text_document_classification_20newsgroups.py`
```

Comment:
What's happening here?

Reply:
I've had issues with `twenty_newsgroup.rst`. Some changes happened in the file after I started the PR, and I kept using the older version.
I thought I had catch all the changes, but apparently not. It should be ok now.